skuber/src/test/scala/com/virtuslab/iat/skubertest/SkuberLabelExpressionsTest.scala

```diff
@@ -6,6 +6,8 @@ import org.scalatest.flatspec.AnyFlatSpec
 import org.scalatest.matchers.should.Matchers
 import skuber.LabelSelector._
 
+import scala.language.reflectiveCalls
```

Comment:
where is the reflective call?

Reply:
`is`, `in` etc. on anynomous class declared in `com.virtuslab.iat.dsl.ExpressionsOps#stringToExpression`; fixed now (the class in question is no longer anonymous)